cmd/kyverno/main.go

```diff
@@ -180,6 +180,8 @@ func main() {
 		maxQueuedEvents              int
 		omitEvents                   string
 		autoUpdateWebhooks           bool
+		disallowValidateWebhook      bool
```

Comment:
We can have single argument that disables all webhook create/updates.

Reply:
Okay, I will do that. I thought we would want to disable selectively.